### DISCUSSION TITLE
Tags: searching, querying.

### DISCUSSION THREAD

**Discussion by avilesj** [nvim-neorg/neorg#1410] [Q&A]
I understand we can tag as `#mytag val1 val2`. Do we have a way to query all workspace files looking for tags? Looking to jump to Neorg from Logseq.

**Comment by celfred** [comment]
Hello,
I wish this question had an answer since I have the same question :)

I was wondering if this could be linked with the 'category' lists we can add in metadata. Would there be a way to auto-complete them by querying all used keywords in a workspace ?

Thanks !

**Comment by rob32** [comment]
I'm also on the fence about tags and neorg (also ex logseq user here). 

Would be cool to know how other neorg users use tags, if at all?

**Reply by Anrock** [reply]
I don't use tags at all since it seems you can't do anything with them except writing them down.
